src/jit/importer.cpp

```diff
@@ -5757,8 +5776,35 @@ GenTreePtr Compiler::impImportStaticFieldAccess(CORINFO_RESOLVED_TOKEN* pResolve
                                     new (this, GT_CNS_INT) GenTreeIntCon(TYP_INT, pFieldInfo->offset, fs));
             }
             break;
+        }
+        case CORINFO_FIELD_STATIC_READYTORUN_HELPER:
```

Comment:
I had great success with adding `case CORINFO_FIELD_STATIC_READYTORUN_HELPER` to the two places in this file that have `case CORINFO_FIELD_STATIC_SHARED_STATIC_HELPER`. (There's a comment around them that says `We may be able to inline the field accessors in specific instantiations of generic`).

It made RyuJIT stop asking weird questions in inlining scenarios.